include/geode/mesh/core/regular_grid.h

```diff
@@ -101,59 +98,65 @@ namespace geode
 
         const Point< dimension >& origin() const;
 
+        AttributeManager& cell_attribute_manager() const;

```

Comment:
Why moving them? Most of the classes attribute managers are at the bottom. But why not.

Reply:
I just felt that theses accessors to important features should be visible before the functions computing some stuff on the class.